satellite/satellitedb/testdata/postgres.v9.sql

```diff
@@ -0,0 +1,232 @@
+-- AUTOGENERATED BY gopkg.in/spacemonkeygo/dbx.v1
```

Comment:
it is confusing these say "autogenerated" when they aren't

Reply:
techincally I copy pasted it from the schema that dbx generates :D, but I'll replace them